packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -272,7 +271,7 @@ function transform(root: IRElement, codeGen: CodeGen, state: State): t.Expressio
     }
 
     function computeAttrValue(attr: IRAttribute, element: IRElement): t.Expression {
-        const { namespaceURI, tagName } = element.__original as parse5.AST.Default.Element;
+        const { namespaceURI, tagName } = element.__original;
```

Comment:
Any particular reason for this change? I noticed that elsewhere you added an `__original as parse5.AST.Default.Element` rather than removing it.